docs/walkthroughs/create-mobile-client-build.adoc

```diff
@@ -77,7 +77,15 @@ Add configuration relating to the type of build being created e.g debug or relea
 
 ===== Release Build Type
 
-The following additional information is required for an Android Release build:
+As a release build will need to be signed, you need to specify the keystore and private key passwords.
+
+===== Keystore located in source code
+If your keystore is checked into your source code, you will need to take the password value in your build script as an environmental variable. This will allow
+you to set this environmental variable as part of your client build from within the OpenShift UI. This can be done directly when creating the mobile client build or afterwards by editing the build config.
```

Comment:
I don't think we have support for editing environment variables after a build config has been created. This will need to be done through Jenkins itself.

Reply:
I tried it out when testing if setting env vars in the build config would work. Seemed to work. The env vars are exposed to the Job